cypress/integration/sitewide-banner.js

```diff
@@ -86,26 +118,41 @@ describe('Site Wide Banner', () => {
   it('The Site Wide Banner CTA URL is correct for an unauthenticated user', () => {
     cy.anonVisitCampaign(exampleCampaign);
 
-    cy.findByTestId('sitewide-banner-button').should('have.length', 1);
-    cy.findByTestId('sitewide-banner-button').should(
-      'have.attr',
-      'href',
-      'https://vote.dosomething.org/?r=source:web,source_details:hellobar',
-    );
+    if ('#banner-portal > .wrapper > [data-test=site-wide-banner]') {
+      cy.findByTestId('sitewide-banner-button').should('have.length', 1);
+      cy.findByTestId('sitewide-banner-button').should(
+        'have.attr',
+        'href',
+        'https://vote.dosomething.org/?r=source:web,source_details:hellobar',
+      );
+    }
```

Comment:
Let's remove this conditional as well. We'd always expect the banner in this scenario.
```suggestion
    cy.findByTestId('sitewide-banner-button').should('have.length', 1);
    cy.findByTestId('sitewide-banner-button').should(
      'have.attr',
      'href',
      'https://vote.dosomething.org/?r=source:web,source_details:hellobar',
    );
```